src/elements/test/DropdownSelect.test.tsx

```diff
@@ -0,0 +1,125 @@
+import { render, screen } from "@testing-library/react";
```

Comment:
Could use the `renderWithContext` function here instead of `render` to make the `useTheme` hook work.

Reply:
wait a minute... I think i know what you where you were going with that ^, i will change it and run the tests again

Reply:
Yeah, I thought about splitting them into a Redux context and theme context function, but ended up just combining them into a single function.